README.md

```diff
@@ -58,6 +58,7 @@ a substring) or a RegExp (the line should match the expression).
 ### function wait (expectation)
```

Comment:
callback missing from this line as an arg, I assume its the second arg, not an alternative for the first arg?